hack/test-go.sh

```diff
@@ -204,7 +207,11 @@ reportCoverageToCoveralls() {
 IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
 for apiVersion in "${apiVersions[@]}"; do
   echo "Running tests for APIVersion: $apiVersion"
-  KUBE_API_VERSION="${apiVersion}" runTests "$@"
+  KUBE_API_VERSION="${apiVersion}"
+  echo "Using standard etcd path prefix: ${ETCD_STANDARD_PREFIX}"
+  ETCD_PREFIX=${ETCD_STANDARD_PREFIX} runTests "$@"
+  echo "Using custom etcd path prefix: ${ETCD_CUSTOM_PREFIX}"
+  ETCD_PREFIX=${ETCD_CUSTOM_PREFIX} runTests "$@"
```

Comment:
What do you think about running this after the for loop instead of during the for loop? (with the most recent api version.) That way we go from two runs on every submit to three, instead of four.